src/System.Management.Automation/engine/ExecutionContext.cs

```diff
@@ -1194,12 +1159,8 @@ internal TypeInfoDataBaseManager FormatDBManager
                 return _formatDBManager;
             }
 
-            // This needs to exist so that RunspaceConfiguration can
-            // push it's shared format database table into ExecutionContext
             set
             {
-                if (this.RunspaceConfiguration != null)
-                    throw new NotImplementedException("set_FormatDBManager()");
                 _formatDBManager = value;
```

Comment:
It looks to me we can remove the set method now, as it's only used for a RunspaceConfiguration scenario.

Reply:
will remove

Reply:
looks like InitialSessionState.cs:3582 uses it